src/commands/lambda/flare.ts

```diff
@@ -166,8 +170,58 @@ export class LambdaFlareCommand extends Command {
     const configStr = util.inspect(config, false, undefined, true)
     this.context.stdout.write(`\n${configStr}\n`)
 
+    // Get project files
+    this.context.stdout.write(chalk.bold('\n📁 Searching for project files in current directory...\n'))
+    const projectFilesToPath = await getProjectFiles()
+    let projectFilesMessage = chalk.bold('\n✅ Found project file(s):\n')
+    if (projectFilesToPath.size === 0) {
+      projectFilesMessage = commonRenderer.renderSoftWarning('No project files found.')
+    }
+    this.context.stdout.write(projectFilesMessage)
+    for (const projectFilePath of projectFilesToPath.values()) {
+      this.context.stdout.write(`• ${projectFilePath}\n`)
+    }
+
+    // Additional files
+    this.context.stdout.write('\n')
+    const additionalFiles = new Set<string>()
+    const addFilesQuestion = await inquirer.prompt(
+      confirmationQuestion('Do you want to specify any additional files to send to Datadog support?')
+    )
```

Comment:
Why are we prompting in the main file when we have the `prompt.ts`?

Reply:
Also, if I press enter here, it defaults to `Yes`. Is this ideal?

Reply:
You're right, I can use the `requestChangesConfirmation` helper method in `prompt.ts`.
I have renamed this function to `requestConfirmation` which better suits its multiple uses now.

Reply:
No, I think that these confirmation questions should default to `No` if enter was pressed. To achieve this change, I have updated: `confirmationQuestion()` to have the line: `default: false,`.

Although, this will change this behavior in `instrument.ts` and `uninstrument.ts` when asking the user 'Do you want to apply the changes?'

Is that okay? WDYT?

Reply:
No, a breaking change is never ok when we are working on something that is not relevant to that component.

We want to maintain previous behaviors and reuse the same methods.

Reply:
Understood. I have updated the code to accept an optional boolean parameter that defaults to true as to not affect instrument and uninstrument.
```
export const confirmationQuestion = (message: string, defaultValue = true): inquirer.ConfirmQuestion => ({
  message,
  name: 'confirmation',
  type: 'confirm',
  default: defaultValue,
})
```